tests.js

```diff
@@ -5079,11 +5058,100 @@ window.Specs = {
 	},
 
 	"css-counter-styles-3": {
-		"title": "CSS Counter Styles Level 3",
+		"title": "CSS Lists and Counters Module Level 3",
 		"links": {
 			"tr": "css-counter-styles-3",
 			"dev": "css-counter-styles-3"
 		},
+		"values": {
+			"properties": [
+				"list-style-type"
+			],
+			"Simple: Numeric": {
+				"links": {
+					"tr": "#simple-numeric",
+					"dev": "#simple-numeric",
+					"mdn": "list-style-type"
+				},
+				"tests": [
+					"decimal", "decimal-leading-zero", "arabic-indic", "armenian", "upper-armenian", "lower-armenian",
+					"bengali", "cambodian", "khmer", "cjk-decimal", "devanagari", "georgian", "gujarati", "gurmukhi",
+					"hebrew", "kannada", "lao", "malayalam", "mongolian", "myanmar", "oriya", "persian", "lower-roman",
+					"upper-roman", "tamil", "telugu", "thai", "tibetan"
+				]
+			},
+			"Simple: Alphabetic": {
+				"links": {
+					"tr": "#simple-alphabetic",
+					"dev": "#simple-alphabetic",
+					"mdn": "list-style-type"
+				},
+				"tests": [
+					"lower-alpha", "lower-latin", "upper-alpha", "upper-latin", "lower-greek", "hiragana",
+					"hiragana-iroha", "katakana", "katakana-iroha"
+				]
+			},
+			"Simple: Symbolic": {
+				"links": {
+					"tr": "#simple-symbolic",
+					"dev": "#simple-symbolic",
+					"mdn": "list-style-type"
+				},
+				"tests": [
+					"disclosure-open", "disclosure-closed"
+				]
+			},
+			"Simple: Fixed": {
+				"links": {
+					"tr": "#simple-fixed",
+					"dev": "#simple-fixed",
+					"mdn": "list-style-type"
+				},
+				"tests": [
+					"dcjk-earthly-branch", "cjk-heavenly-stem"
+				]
+			},
+			"Complex: Japanese": {
+				"links": {
+					"tr": "#limited-japanese",
+					"dev": "#limited-japanese",
+					"mdn": "list-style-type"
+				},
+				"tests": [
+					"japanese-informal", "japanese-formal"
+				]
+			},
+			"Complex: Korean": {
+				"links": {
+					"tr": "#limited-korean",
+					"dev": "#limited-korean",
+					"mdn": "list-style-type"
+				},
+				"tests": [
+					"korean-hangul-formal", "korean-hanja-informal", "korean-hanja-formal"
+				]
+			},
+			"Complex: Chinese": {
+				"links": {
+					"tr": "#limited-chinese",
+					"dev": "#limited-chinese",
+					"mdn": "list-style-type"
+				},
+				"tests": [
+					"simp-chinese-informal", "simp-chinese-formal", "trad-chinese-informal", "trad-chinese-formal"
```

Comment:
As it is in the specification and browsers may still implement it, I think you should add [`cjk-ideographic`](https://drafts.csswg.org/css-counter-styles-3/#cjk-ideographic), even when it's deprecated.

Reply:
Ok, I add it.